### ISSUE TITLE
docs: document EverywhereReplicationStrategy

### ISSUE THREAD

**Issue by annastuchlik** [scylladb/scylladb#12417]
In issue https://github.com/scylladb/scylla-doc-issues/issues/823 we recommend using `EverywhereReplicationStrategy`. Also, that strategy is mentioned in other issues on GitHub, but it is not documented.

Please provide me with information about `EverywhereReplicationStrategy` so that I can add its description to https://docs.scylladb.com/stable/cql/ddl.html.

@vladzcloudius @kbr-scylla @tzach @avikivity Could you help?

**Comment by vladzcloudius** [comment]
@avikivity Is there anything beyond "This is a replication strategy that will replicate on all nodes in the cluster." to say about this strategy?

I believe it is also going to  adjust the effective RF as nodes are added/removed to/from the cluster.

**Comment by kbr-scylla** [comment]
> I believe it is also going to adjust the effective RF as nodes are added/removed to/from the cluster.

That's right, the number of replicas is always equal to the number of token ring members.

`EverywhereReplicationStrategy` has no sub-options, there is not much to say about it other than that it replicates to all nodes.

Perhaps we can write a recommendation such as: use it only for small tables with rare operations (e.g. storing some metadata which is rarely updated) because all nodes in the cluster store a copy of the data and all nodes may participate in the operations (which may be costly in a 100 node cluster, for example).
